### PR TITLE
autoconf: complete the PKG family

### PR DESCRIPTION
**Pull request by jannick0**
Hi again,

@ptomato In this PR I adjusted one file only as last time - I was wildly guessing that `Autoconf.JSON-tmLanguage` was the correct one. I hope that the build process updates the other files.  I did my best to insert the new items into the right categories.

@maelvls Would you be able to pull this PR - if correct - into `vscode_autotools`, too?

Using `vscode_autotools` which is based on this repo saves me so much headaches about better `autoconf` speach, embedded shell code and bracketing where one could get easily lost otherwise.  Many thanks to both of you!

### CODE REVIEW

File: Autoconf.JSON-tmLanguage [hunk 1]
```diff
@@ -424,7 +424,7 @@
...
...           "begin": "\\b(AC_(?:CHECK_(?:DECL|FUNC|HEADER)S_ONCE|CHECKING|CONFIG_(?:(?:(?:AUX|LIBOBJ)_|SRC)DIR|MACRO_DIRS?|SUBDIRS)|COPYRIGHT|ERLANG_NEED_ERLC?|F(?:77|C)_LIBRARY_LDFLAGS|FATAL|INCLUDES_DEFAULT|LANG(?:_(?:ASSERT|CONFTEST|FUNC_LINK_TRY|POP|PUSH|SOURCE))?|LIB(?:OBJS|SOURCES?)|MSG_(?:CHECKING|NOTICE|RESULT|WARN)|OPENMP|PREFIX_(?:DEFAULT|PROGRAM)|PREREQ|PROG_(?:CC|CXX|F77|OBJC(?:XX)?)|REPLACE_FUNCS|REQUIRE(?:_AUX_FILE)?|REVISION|SUBST_FILE|WARNING)|AH_(?:BOTTOM|TOP)|AS_(?:DIRNAME|(?:EXECUTABLE|MKDIR)_P|EXIT|SET_STATUS|TR_(?:CPP|SH)|VAR_(?:POPDEF|TEST_SET))|m4_(?:chomp(?:_all)?|cleardivert|debug(?:file|mode)|decr|defn|divert(?:_(?:pop|push))?|errprintn|(?:re_)?escape|exit|expand|flatten|s?include|incr|len|(?:make|mks)temp|n|newline|normalize|pattern_(?:allow|forbid)|popdef|set_(?:delete|list|listc|size)|sign|strip|to(?:lower|upper)|warning|wrap(?:_lifo)?)|AT_(?:BANNER|CAPTURE_FILE|COPYRIGHT|INIT|KEYWORDS|SETUP|TESTED)|AM_(?:GNU_GETTEXT_(?:VERSION|NEED)|EXTRA_RECURSIVE_TARGETS|ICONV|INIT_AUTOMAKE|MAINTAINER_MODE|PROG_UPC|SUBST_NOTMAKE|XGETTEXT_OPTION)|LT_(?:INIT|LANG|PREREQ)|IT_PO_SUBDIR|GOBJECT_INTROSPECTION_(?:CHECK|REQUIRE)|PKG_PREREQ|PKG_(NOARCH_)?INSTALLDIR)\\s*(\\()",
```
Comment (ptomato) [inline Autoconf.JSON-tmLanguage]: This should be `(?:NOARCH_)` so it doesn't form a capturing group.

File: Autoconf.JSON-tmLanguage [hunk 2]
```diff
@@ -589,7 +589,7 @@
         },
 
         "func-mmss": {
-            "begin": "\\b(AC_(?:ARG_(?:ENABLE|WITH)|EGREP_(?:CPP|HEADER))|AS_VAR_IF|AT_ARG_OPTION(?:_ARG)?|PKG_CHECK_MODULES)\\s*(\\()",
+            "begin": "\\b(AC_(?:ARG_(?:ENABLE|WITH)|EGREP_(?:CPP|HEADER))|AS_VAR_IF|AT_ARG_OPTION(?:_ARG)?|PKG_CHECK_MODULES(_STATIC)?)\\s*(\\()",
```
Comment (ptomato) [inline Autoconf.JSON-tmLanguage]: Likewise, this should be `(?:_STATIC)`.

Comment (jannick0) [reply Autoconf.JSON-tmLanguage]: Done.

**Comment by jannick0** [comment]
Both done in the latest push. Thanks for the swift reply!

**Comment by maelvls** [comment]
I'll pull now, sorry for the delay 😄

**Comment by maelvls** [comment]
Hey. I have released the version v0.2.0 for VSCode. It is now up-to-date with the latest changes from ptomato/sublime_autotools, including this fix.

**Pull request merged**